slothy/helper.py

```diff
@@ -33,11 +33,19 @@ class SourceLine:
     """Representation of a single line of source code"""
 
     def _extract_comments_from_text(self):
-        if not "//" in self._raw:
+        # Match // and /* ... */ style comments with optionally preceeding code
```

Comment:
This is a slight change of behaviour for lines of the form `text // comment A //comment B`, isn't it? Previously, we would have extracted `comment A` and `comment B` separately. Can you retain that?

Reply:
@hanno-becker Thanks for pointing this out. 
Commit b9e5a805e023f2e066973e8d010201bdbfca66c7 should preserve the original behaviour. 
Find some test cases I ran below: 
```
Text: 'abc // def // ghi', Code: 'abc', Comments: ['def', 'ghi']
Text: 'abc /* def */ /* ghi */', Code: 'abc', Comments: ['def', 'ghi']
Text: 'just code', Code: 'just code', Comments: []
Text: 'just code//', Code: 'just code', Comments: ['']
Text: '// just comment', Code: '', Comments: ['just comment']
Text: '/* just comment */', Code: '', Comments: ['just comment']
Text: 'abc // mixed /* comment */ // styles', Code: 'abc', Comments: ['mixed', 'comment', 'styles']
Text: 'abc /* mixed // comment styles */', Code: 'abc', Comments: ['mixed // comment styles']
Text: 'abc // /* more mixing */', Code: 'abc', Comments: ['', 'more mixing']
``` 

Note that empty comments will be stripped away in `_trim_comments()` anyways.

Reply:
Thanks @dop-amin! I think this would be easier to read if the code handled `/* ... */`-style comments and `// ...`-style comments separately. E.g. you could first handle all `/* ... */` comments and then use the previous code for `// ...`.